README.md

```diff
@@ -50,15 +50,8 @@ To upload results to the Security tab of your repo, run the `github/codeql-actio
 
 ## Advanced
 
-To configure **Container Mapping** to send to **Microsoft Defender for DevOps**, include `container-mapping` as a tool:
-```yaml
-- uses: microsoft/security-devops-action@v1
-  id: msdo
-  with:
-    includeTools: container-mapping
```

Comment:
`includeTools` was added primarily to add the tools in-addition to the tools set by policies. One of the main use cases of the MSDO task was to enable policy-based tooling and `includeTools` (and in extension `excludeTools`) was a way to give the developer a way to modify these policies set by external members (like the CISO). 

I would recommend not removing it, and also instead adding the `excludeTools` functionality given that we're enabling container mapping by default

Reply:
Discussed offline. `includeTools` only ever had an effect on container-mapping, but this change pre-empts that effect. The `tools` command is now the expected way to configure specific tools.